pyaedt/modules/Boundary.py

```diff
@@ -643,146 +847,140 @@ def override_component(
         bool
             ``True`` if successful. ``False`` otherwise.
         """
-        if self.props["NativeComponentDefinitionProvider"]["PartsChoice"] != 1:
+        if self._app.settings.aedt_version < "2024.2":
             self._app.logger.error(
-                "Device Parts modeling is not active, hence no filtering or override option is available."
+                "This method is available only with AEDT 24R2 or newer. Use override_instance instead."
```

Comment:
```suggestion
                "This method is available only with AEDT 2024 R2 or later. Use 'override_instance()' method instead."
```
User-facing doc should use this release format: 2024 Set of method name in single quotation given that double backticks can't be used.